lib/src/tabs/Tab.tsx

```diff
@@ -0,0 +1,181 @@
+import React, { forwardRef, Ref } from "react";
+import styled from "styled-components";
+import { TabProps } from "./types";
+import DxcBadge from "../badge/Badge";
+
+const Tab = forwardRef(
+  (
+    { active, tab, tabIndex, hasLabelAndIcon, iconPosition, onClick, onMouseEnter, onMouseLeave }: TabProps,
+    ref: Ref<HTMLButtonElement>
+  ): JSX.Element => {
+    return (
+      <TabContainer
+        role="tab"
+        type="button"
+        tabIndex={tabIndex}
+        disabled={tab.isDisabled}
+        aria-disabled={tab.isDisabled}
+        aria-selected={active}
+        hasLabelAndIcon={hasLabelAndIcon}
+        iconPosition={iconPosition}
+        ref={ref}
+        onClick={() => {
+          onClick();
+        }}
+        onMouseEnter={() => {
+          onMouseEnter();
+        }}
+        onMouseLeave={() => {
+          onMouseLeave();
+        }}
+      >
+        <MainLabelContainer
+          hasBadge={tab.notificationNumber}
+          hasLabelAndIcon={hasLabelAndIcon}
+          iconPosition={iconPosition}
+        >
+          {tab.icon && (
+            <TabIconContainer hasLabelAndIcon={hasLabelAndIcon} iconPosition={iconPosition}>
+              {typeof tab.icon === "string" ? <TabIcon src={tab.icon} /> : tab.icon}
+            </TabIconContainer>
+          )}
+          <LabelTextContainer>{tab.label}</LabelTextContainer>
+        </MainLabelContainer>
+        {tab.notificationNumber && tab.notificationNumber && (
+          <BadgeContainer hasLabelAndIcon={hasLabelAndIcon} iconPosition={iconPosition}>
+            <DxcBadge notificationText={tab.notificationNumber > 99 ? "+99" : tab.notificationNumber} />
+          </BadgeContainer>
+        )}
+      </TabContainer>
+    );
+  }
+);
+
+type IconProps = {
+  hasLabelAndIcon: boolean;
+  iconPosition: "top" | "left";
+};
+
+const TabContainer = styled.button<IconProps>`
+  text-transform: ${(props) => props.theme.fontTextTransform};
+  overflow: hidden;
+  text-align: center;
+  flex-shrink: 0;
+  line-height: 1.75;
+  white-space: normal;
+  letter-spacing: 0.02857em;
+  border: 0;
+  cursor: pointer;
+  display: inline-flex;
+  align-items: center;
+  user-select: none;
+  vertical-align: middle;
+  justify-content: center;
+  font-family: ${(props) => props.theme.fontFamily};
+  font-size: ${(props) => props.theme.fontSize};
+  font-style: ${(props) => props.theme.fontStyle};
+  font-weight: ${(props) => props.theme.fontWeight};
+  min-width: 90px;
+  max-width: 360px;
+  padding: ${(props) =>
+    ((!props.hasLabelAndIcon || (props.hasLabelAndIcon && props.iconPosition !== "top")) && "12px 16px") || "8px 16px"};
+  height: ${(props) =>
+    ((!props.hasLabelAndIcon || (props.hasLabelAndIcon && props.iconPosition !== "top")) && "47px") || "71px"};
+  min-height: ${(props) =>
+    ((!props.hasLabelAndIcon || (props.hasLabelAndIcon && props.iconPosition !== "top")) && "47px") || "71px"};
+  background-color: ${(props) => props.theme.unselectedBackgroundColor};
```

Comment:
Height and min height values are `48px` and `72px`, instead of `47px` and `71px` as here.

Reply:
Yes I know, this is on purpose. Because the underline is on top of the tabs so if we higher the height, the tabs will be covering the underline.
With height 47px:
![image](https://user-images.githubusercontent.com/34685461/190618188-f1e59c22-2ebe-4989-afcd-826200da554b.png)

With height 48px:
![image](https://user-images.githubusercontent.com/34685461/190618092-3e7059b5-f389-44a2-acce-6122adce7922.png)

I have been trying other solutions like changing the z-index of the underline to 2 and height of the tabs to 48px but this happens:
![image](https://user-images.githubusercontent.com/34685461/190618516-3d795fa8-ea4b-4f23-863b-3c71681c6029.png)
The underline will be covering the 1px that belongs to the outline focus of the tab.

However, because the underline is 1px, in the end the whole tabs with the underline will be 48px.
![image](https://user-images.githubusercontent.com/34685461/190619414-034e656a-5969-40af-af10-89dde44bf1c7.png)